homeassistant/components/oralb/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "domain": "oralb",
+  "name": "OralB",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/oralb",
+  "bluetooth": [
+    {
+      "service_uuid": "0000fe0d-0000-1000-8000-00805f9b34fb"
+    }
+  ],
+  "requirements": ["oralb==0.1.2"],
```

Comment:
[There is a `sleep` in async code.](https://github.com/bkbilly/OralB/blob/main/oralb/__init__.py#L116) This will block all of Home Assistant for 3s when it executes. This should be removed.

Reply:
Removed the sleep.